src/System.Runtime.Extensions/tests/System/Runtime/ProfileOptimization.netcoreapp.cs

```diff
@@ -5,22 +5,36 @@
 using System.Diagnostics;
 using System.IO;
 using Xunit;
+using Xunit.Abstractions;
 
 namespace System.Runtime.Tests
 {
     public class ProfileOptimizationTest : RemoteExecutorTestBase
     {
+        private readonly ITestOutputHelper _output;
+
+        public ProfileOptimizationTest(ITestOutputHelper output) => _output = output;
+
         [Fact]
         public void ProfileOptimization_CheckFileExists()
         {
             string tmpProfileFilePath = GetTestFileName();
+            string tmpTestFileName = Path.Combine(Path.GetDirectoryName(tmpProfileFilePath), Path.GetRandomFileName());
+
+            _output.WriteLine($"We'll test write permission on path '{tmpTestFileName}'");
```

Comment:
@brianrob @danmosemsft  added some console logs using xunit helper...it doesn't work on my local...is this expected?If so do i use `Console.WriteLine`?

Reply:
ping @Anipik @brianrob @danmosemsft